src/components/Card/index.examples.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import {storiesOf} from '@kadira/storybook'
+import Card from '.'
+
+storiesOf('Static: Cards')
+  .addWithInfo('Course Card', () => (
+    <div className='flex justify-center items-center bg-dark-navy pv5'>
```

Comment:
These styles don't refer to the actual component, it is for the represent components inside storybook (black background, aligning and so on). Make sense?